src/components/List/index.js

```diff
@@ -43,15 +47,25 @@ export default function List (props) {
       options={categories}
       selected={category}
     />
-    <ListGroup className="mb-3">
-      {filteredDataByCategory.map(listing => <Listing
-        category={listing.category}
-        description={listing.details}
-        key={listing.location}
-        location={listing.location}
-        type={typeConvert[listing.type]}
-      />)}
-    </ListGroup>
+    {
+      !filteredDataByCategory.length &&
+      <Message
+        spinner={false}
```

Comment:
You don't need a `false` boolean literal either because it's absence signals being false-y.
```suggestion
```